web/packages/teleterm/src/mainProcess/rootClusterProxyHostAllowList.ts

```diff
@@ -77,22 +77,24 @@ export function manageRootClusterProxyHostAllowList({
 
     allowList.clear();
     for (const rootCluster of rootClusters) {
-      if (!rootCluster.proxyHost) {
-        continue;
+      if (rootCluster.proxyHost) {
+        let browserProxyHost: string;
+        try {
+          browserProxyHost = proxyHostToBrowserProxyHost(rootCluster.proxyHost);
+        } catch (error) {
+          logger.error(
+            'Ran into an error when converting proxy host to browser proxy host',
+            error
+          );
+          continue;
+        }
+        allowList.add(browserProxyHost);
```

Comment:
I think we should remove `continue` and move `allowList.add(browserProxyHost)` to `try { }`.

IMO there's no need to skip adding `ssoHostname` if adding `browserProxyHost` failed.